### PR TITLE
Issue1014 dymola2019 fd01 beta1

### LINKED ISSUE #1014: correct issues that came up with Dymola 2019FD01 beta1 testing
This branch is to correct various small issues that came up in testing.

### PR DESCRIPTION
**Pull request by mwetter**
This closes #1014

### CODE REVIEW

File: IBPSA/Fluid/Sensors/BaseClasses/PartialAbsoluteSensor.mo [hunk 1]
```diff
@@ -29,6 +29,17 @@ IBPSA.Fluid.Sensors.BaseClasses.PartialFlowSensor</a>.
 revisions="<html>
 <ul>
 <li>
+September 7, 2019, by Michael Wetter:<br/>
```
Comment (Mathadon) [inline IBPSA/Fluid/Sensors/BaseClasses/PartialAbsoluteSensor.mo]: 2018 =)

File: IBPSA/Fluid/Sensors/BaseClasses/PartialAbsoluteSensor.mo [hunk 2]
```diff
@@ -29,6 +29,17 @@ IBPSA.Fluid.Sensors.BaseClasses.PartialFlowSensor</a>.
...
+Changed
+<code>port(redeclare package Medium=Medium, m_flow(min=0, max=0))</code>
+to
+<code>port(redeclare package Medium=Medium, m_flow(min=0))</code>
+to avoid in Dymola 2019FD01 beta1 the message
+\"port.m_flow has the range [0,0] - which is suspicious since the max-value should be above the min-value\"
```
Comment (Mathadon) [inline IBPSA/Fluid/Sensors/BaseClasses/PartialAbsoluteSensor.mo]: What a lame error! max=min=0 is completely reasonable. I'll report this.

**Comment by Mathadon** [comment]
@mwetter this can be merged when unit tests finish

File: IBPSA/Fluid/Sensors/BaseClasses/PartialAbsoluteSensor.mo (hunk 2 above)
Comment (mwetter) [reply IBPSA/Fluid/Sensors/BaseClasses/PartialAbsoluteSensor.mo]: I already reported it to the Dymola developers. I think this is a corner case as Modelica does not allow comparing reals for equality due to rounding errors, and max m_flow =min m_flow =0 implies m_flow == 0 which is not allowed as a test.
